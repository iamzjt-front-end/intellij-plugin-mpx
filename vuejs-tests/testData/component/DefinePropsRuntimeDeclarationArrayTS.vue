<script setup lang="ts">
defineProps(["optional", "required"]);
</script>

<template>
  {{optional}}
  {{required}}
</template>
