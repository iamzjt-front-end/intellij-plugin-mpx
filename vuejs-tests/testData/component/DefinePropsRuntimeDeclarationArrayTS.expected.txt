{
 class: "VueSourceComponent",
 components: {
  DefinePropsRuntimeDeclarationArrayTS: <recursive value of class VueSourceComponent>,
 },
 props: [
  {
   class: "VueSourceInputProperty",
   name: "optional",
   required: false,
  },
  {
   class: "VueSourceInputProperty",
   name: "required",
   required: false,
  },
 ],
}
